runpod/serverless/worker.py

```diff
@@ -49,53 +49,77 @@ def _is_local(config) -> bool:
 # ------------------------- Main Worker Running Loop ------------------------- #
 async def run_worker(config: Dict[str, Any]) -> None:
     """
-    Starts the worker loop.
+    Starts the worker loop for multi-processing.
+
+    Args:
+        config (Dict[str, Any]): Configuration parameters for the worker.
     """
     auth_header = _get_auth_header()
-    async with aiohttp.ClientSession(headers=auth_header, timeout=_TIMEOUT) as session:
+    connector = aiohttp.TCPConnector(limit=None)
+
+    async with aiohttp.ClientSession(
+            connector=connector, headers=auth_header, timeout=_TIMEOUT) as session:
 
         heartbeat.start_ping()
 
-        kill_worker = False # Flag to kill the worker after job is complete.
-        while kill_worker is False:
-            job = await get_job(session)
+        # Flag to kill the worker after job is complete.
+        job_scaler = JobScaler(
+            handler_fully_utilized=config.get('handler_fully_utilized'),
```

Comment:
We will test with `handler_fully_utilized` for now but am not sold on the name yet for the final release.

Reply:
Agreed, don't like the name either.